Set protractor environment

michelesr opened the pull request:
Add protractor example tests and setup docker-compose to run selenium and protractor containers.

To run tests:

$ docker-compose run protractor

Image from [protractor-docker](https://registry.hub.docker.com/u/michelesr/protractor/)

michelesr committed:
Set protractor environment

kobe25 commented:
Il `Dockerfile` di `protractor-docker` andrebbe rifatto partendo da `iojs:2.3` o `node:0.12`.  A titolo di esempio vedi `ui/Dockerfile`.

michelesr committed:
Setup e2e test environment

Makefile: add e2e tests

Moved ui/test/protractor -> test/e2e

docker-compose.yml: renamed protractor to e2e and update mount path
according to above move

michelesr commented:
Updated with b5080b4, and also update the image in docker-hub to use `iojs` as base
